### PR TITLE
feat(images): Adding iOS Image Cache Limits

### PR DESCRIPTION
**Pull request by gedeagas**
By default React Native would set `NSCache.totalCostLimit` to 20MB and the single image size limit to 2MB (bitmap size). The default cache size may not be enough for applications that make heavy use of images. 

But thanks to this [commit](https://github.com/facebook/react-native/commit/61b013e7ad8a1cc28ee39434d2fd96b74b96cf5f) now you can configure the image cache size on iOS by adding `RCTSetImageCacheLimits` in your app App Delegate.

This feature should be available from [v0.69.0-rc.0](https://github.com/facebook/react-native/releases/tag/v0.69.0-rc.0)

### CODE REVIEW

**Comment by gedeagas** [comment]
PR Reference 
https://github.com/facebook/react-native/pull/33554

**Comment by gedeagas** [comment]
@Simek since this method is already available since [v0.69.0-rc.0](https://github.com/facebook/react-native/releases/tag/v0.69.0-rc.0) should i also edit the versioned docs?

File: docs/images.md [hunk 1]
```diff
@@ -229,3 +229,18 @@ Please note that the following corner specific, border radius style properties m
...
+```
+
+**Parameters:**
+
+| Name           | Type   | Required | Description             |
+| -------------- | ------ | -------- | ----------------------- |
+| imageSizeLimit | number | Yes      | Image cache size limit. |
+| totalCostLimit | number | Yes      | Total cache cost limit. |
+
+Based on the code above the single image size limit would be set to 4 MB and the total cost limit would be set to 200 MB.
```
Comment (huntie) [inline docs/images.md:246]:
```suggestion
In the above code example the image size limit is set to 4 MB and the total cost limit is set to 200 MB.
```

File: docs/images.md [hunk 2]
```diff
@@ -229,3 +229,18 @@ Please note that the following corner specific, border radius style properties m
 ## Off-thread Decoding
 
 Image decoding can take more than a frame-worth of time. This is one of the major sources of frame drops on the web because decoding is done in the main thread. In React Native, image decoding is done in a different thread. In practice, you already need to handle the case when the image is not downloaded yet, so displaying the placeholder for a few more frames while it is decoding does not require any code change.
+
+## Configuring iOS Image Cache Limits
+
```
Comment (huntie) [inline docs/images.md:234]: Ideally we can include 1-2 sentences of context here. Roughly:

> On iOS, we expose an API to override React Native's default image cache limits. This should be called from within your native `AppDelegate` code (e.g. within (e.g. within `didFinishLaunchingWithOptions`).

**Comment by gedeagas** [comment]
@huntie updated the docs, please kindly check. thank you

File: docs/images.md [hunk 3]
```diff
@@ -232,6 +232,8 @@ Image decoding can take more than a frame-worth of time. This is one of the majo
 
 ## Configuring iOS Image Cache Limits
 
+On iOS, we expose an API to override React Native's default image cache limits. This should be called from within your native AppDelegate code (e.g. within (e.g. within didFinishLaunchingWithOptions).
```
Comment (Simek) [inline docs/images.md:235]:
```suggestion
On iOS, we expose an API to override React Native's default image cache limits. This should be called from within your native AppDelegate code (e.g. within `didFinishLaunchingWithOptions`).
```
Small nit.

**Review by Simek — APPROVED** [review]
Thank you for this update! 👍 Looks good to me beside the small nit I have commented about.

Also, it would be nice to edit the `0.69` docs file, since as you described, the change has landed already in latest release.

**Comment by gedeagas** [comment]
Updated @Simek, please kindly check again. Thank you.

**Review by Simek — APPROVED** [review]
Thank you for all of the updates, LGTM! 👍

**Comment by gedeagas** [comment]
Oke thank you @Simek.

**Pull request merged**